opta/commands/secret.py

```diff
@@ -38,15 +42,14 @@ def view(secret: str, env: Optional[str], config: str) -> None:
     _raise_if_no_k8s_cluster_exists(layer)
 
     configure_kubectl(layer)
-    load_kube_config()
-    v1 = CoreV1Api()
-    api_response = v1.read_namespaced_secret("secret", layer.name)
-    if secret not in api_response.data:
+    create_namespace(layer.name)
+    secrets = deep_merge(get_manual_secrets(layer.name), get_linked_secrets(layer.name))
```

Comment:
Can we just create a `get_secrets` function in kubernetes.py? Instead of doing this merge logic at every place?

Reply:
done